Modules/_ctypes/callproc.c

```diff
@@ -64,6 +64,10 @@
 #include "ctypes_dlfcn.h"
 #endif
 
+#ifdef __APPLE__
+extern bool _dyld_shared_cache_contains_path(const char* path) __attribute__((weak_import));
```

Comment:
This can be replaced with `#include <mach-o/dyld.h>` after Seed 3

Reply:
Can we please wait for official API here?

Reply:
Seed 3 is out today.

Reply:
looks like it got pushed out to seed 4

Reply:
It is in Xcode 12 beta 4, only enough with the same name but with documentation in a header file.

Reply:
yup, updated the pull request to include `mach-o/dyld.h` instead of declaring a weak import